boards/arm/raytac_mdbt53_db_40_nrf5340/doc/index.rst

```diff
@@ -0,0 +1,370 @@
+   ``-DBOARD=raytac_mdbt53_db_40_nrf5340_cpunet``.
+
+
+When building a Secure/Non-Secure application for the nRF5340 application core,
+the Secure application will have to set the IDAU (SPU) configuration to allow
+Non-Secure access to all CPU resources utilized by the Non-Secure application
+firmware. SPU configuration shall take place before jumping to the Non-Secure
+application.
+
+Building a Secure only application
+==================================
+
+Build the Zephyr app in the usual way (see :ref:`build_an_application`
+and :ref:`application_run`), using ``-DBOARD=raytac_mdbt53_db_40_nrf5340_cpuapp`` for
+the firmware running on the nRF5340 application core, and using
+``-DBOARD=raytac_mdbt53_db_40_nrf5340_cpunet`` for the firmware running
+on the nRF5340 network core.
+
+Flashing
+========
+
+Follow the instructions in the :ref:`nordic_segger` page to install
+and configure all the necessary software. Further information can be
+found in :ref:`nordic_segger_flashing`. Then build and flash
+applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+.. warning::
+
+   The nRF5340 has a flash read-back protection feature. When flash read-back
+   protection is active, you will need to recover the chip before reflashing.
+   If you are flashing with :ref:`west <west-build-flash-debug>`, run
+   this command for more details on the related ``--recover`` option:
+
+Here is an example for the :ref:`hello_world` application.
+
+Use a USB to TTL converter to connect the computer and raytac_mdbt53_db_40_nrf5340
+J10 connector. Then run your favorite terminal program to listen for output.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the USB to TTL converter
+can be found. For example, under Linux, :code:`/dev/ttyUSB0`.
+
+Then build and flash the application in the usual way.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: raytac_mdbt53_db_40_nrf5340
+   :goals: build flash
+
+Debugging
+=========
+
+The ``raytac_mdbt53_db_40_nrf5340`` board does not have an on-board-J-Link debug IC,
+however, instructions from the :ref:`nordic_segger` page also apply to this board.
+Use the Debug out connector of nRF52x DK to connect to the J1 connector, and use SEGGER
+J-Link OB IF to debug.
+
+Refer to the :ref:`nordic_segger` page to learn about debugging Nordic
+boards with a Segger IC.
+
+
+Testing the LEDs and buttons in the MDBT53-DB-40
+************************************************
+
+There are 2 samples that allow you to test that the buttons (switches) and LEDs on
+the board are working properly with Zephyr:
+
+   samples/basic/blinky
+   samples/basic/button
+
+You can build and flash the examples to make sure Zephyr is running correctly on
+your board. The button and LED definitions can be found in
+:zephyr_file:`boards/arm/raytac_mdbt53_db_40_nrf5340/raytac_mdbt53_db_40_nrf5340.dts`.
```

Comment:
this information is redundant, basic samples are well known and are expected to work with all boards that have buttons/leds.

Reply:
I have deleted this information.